packages/fxa-auth-db-mysql/db-server/test/backend/remote.js

```diff
@@ -876,7 +874,7 @@ module.exports = function(cfg, makeServer) {
             assert(token.createdAt, 'Got a createdAt')
             assert.equal(!! token.emailVerified, user.account.emailVerified)
             assert(token.verifierSetAt, 'verifierSetAt is set to a truthy value')
-            assert.equal(token.tokenVerificationId, undefined, 'tokenVerificationId is undefined')
+            assert.isUndefined(token.tokenVerificationId)
```

Comment:
How come some places the check is [`isNull`](isNull(token.tokenVerificationId), here it's `isUndefined`? Doesn't need to be addressed for this PR, just seems odd to me.

Reply:
Presumably because his is a `keyFetchToken` and the other one is a `sessionToken`? `tokenVerificationId` is not a property of `keyFetchToken`.